docs/hub/repositories.md

```diff
@@ -0,0 +1,210 @@
+
+In other words, you can commit changes as `Elon Musk <elon@tesla.com>`, push them to your preferred `git` host (for instance github.com) and your commit will link to Elon's GitHub profile. (Try it! But don't blame us if Elon gets mad at you for impersonating him)
+
+See this post by Ale Segala for more context: [How (and why) to sign `git` commits](https://withblue.ink/2020/05/17/how-and-why-to-sign-git-commits.html)
+
+You can prove a commit was authored by you, using GNU Privacy Guard (GPG) and a key server. GPG is a cryptographic tool used to verify the authenticity of a message's origin. To learn about setting up signed commits verification with GPG, visit the [**Security**](TODO) section of our docs!
+
+### Licenses
+
+You are able to add a license to any repo that you create on the Hugging Face Hub to let other users know about the permissions that you want to attribute to your code. Likewise, remember to seek out and respect a project's license if you're considering using their code.
+
+A [**full list of the available licenses**](TODO) is available in these docs.
+
+## Next steps
+
+A good place to visit if you want to continue learning about Git is [GitHub's "Using Git" tutorial](https://docs.github.com/en/get-started/using-git/about-git). These next sections highlight features and additional information that you may find useful to make the most out of the Git repositories on the Hugging Face Hub.
+
+### How to use branches
+
+To effectively use Git repos collaboratively and to work on features without releasing premature code you can use **branches**. Branches allow you to separate your "work in progress" code from your "production-ready" code, with the additional benefit of letting multiple people work on a project without frequently conflicting with each others' contributions. To learn about Git branching, you can try out the [Learn Git Branching interactive tutorial](https://learngitbranching.js.org/).
+
+### Using tags
+
+Git allows you to *tag* commits so that you can easily note milestones in your project. As such, you can use tags to mark commits in your Hub repos! To learn about using tags, you can visit [this DevConnected post](https://devconnected.com/how-to-create-git-tags/).
+
+Beyond making it easy to identify important commits in your repo's history, using Git tags also allows you to [clone a repository at a specific tag](https://www.techiedelight.com/clone-specific-tag-with-git/). The `huggingface_hub` library also supports working with tags, such as [downloading files from a specific tagged commit](https://huggingface.co/docs/huggingface_hub/main/en/how-to-downstream#hfhuburl).
+
+### How to duplicate/fork a repo (including LFS pointers)
+
+A duplicate of a repository is called a *fork*. You may choose to fork one of your own repos, but it also common to fork other people's projects if you would like to tinker with them.
```

Comment:
Duplicate and fork are actually a bit of different concepts. 

Fork = keep all the original git history. This is really a GH concept and although we support it with the list of commands below, it's not a very common flow. This might change at one point with the PRs on the Hub.
Duplicator = duplicate the repo, but without its git history. This is very common for people working with Spaces, as many times you want to get started from another space. Users can use https://huggingface.co/spaces/osanseviero/repo_duplicator for this, so maybe it's worth adding it here.